litecommands-annotations/test/dev/rollczi/litecommands/annotations/async/ParallelAsyncCommandTest.java

```diff
@@ -24,7 +24,6 @@ public class ParallelAsyncCommandTest extends LiteTestSpec {
     private static final int DELAY = 100;
 
     static LiteTestConfig config = builder -> builder
-        .scheduler(new SchedulerExecutorPoolImpl("test"))
```

Comment:
Please revert it, this unit test is used to check a case with the specific scheduler.